lib/utils/messages.ts

```diff
@@ -15,26 +15,28 @@
  * limitations under the License.
  */
 
-const DEBUG_MODE = !!process.env.DEBUG;
-
 export const reachingOut = `\
-If you need help, or just want to say hi, don't hesitate in reaching out
-through our discussion and support forums at https://forums.balena.io
-
-For bug reports or feature requests, have a look at the GitHub issues or
-create a new one at: https://github.com/balena-io/balena-cli/issues/\
+For additional help or support, visit:
+https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
```

Comment:
```
$ ./bin/balena help
....
GLOBAL OPTIONS
  --help, -h
  --debug

For additional help or support, visit:
https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
```
```
$ ./bin/balena login
...
Successfully logged in as: gh_paulo_castro

Find out about the available commands by running:

  $ balena help

For additional help or support, visit:
https://www.balena.io/docs/reference/balena-cli/#support-faq-and-troubleshooting
```

Reply:
The `www.balena.io/docs/reference` link partially reflects the contents of the README file, which is also being modified in this PR, so it would (indirectly) include a link to the masterclasses.